sample-build-scripts/test/xamarin-uitest-android/appcenter-post-build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+echo "Found UI tests projects:"
+find $APPCENTER_SOURCE_DIRECTORY -regex '*.UITest.*\.csproj' -exec echo {} \;
+echo
+echo "Building UI test projects:"
+find $APPCENTER_SOURCE_DIRECTORY -name '*.UITest.csproj' -exec msbuild {} \;
+echo "Compiled projects to run UI tests:"
+find $APPCENTER_SOURCE_DIRECTORY -regex '*.bin.*UITest.*\.dll' -exec echo {} \;
+#echo "Running UI test in App Center Test:"
+echo "What is in the output directory"
+ls $APPCENTER_OUTPUT_DIRECTORY
```

Comment:
Is there a specific reason for listing those files? Would it make more sense to add a check for something and exit if it doesn't work?